src/i18n/messages/all.it.json

```diff
@@ -191,6 +192,7 @@
     "i18nStrings.operatorLessOrEqualText": "Minore o uguale a",
     "i18nStrings.operatorLessText": "Minore di",
     "i18nStrings.operatorStartsWithText": "Inizia con",
+    "i18nStrings.operatorDoesNotStartWithText": "Non inizia con",
```

Comment:
Note: here I manually applied a fix which I am expecting (see `AWSLOC-44490`), in order to unblock #2036 ASAP.